src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -0,0 +1,498 @@
+* **API server**: rate of `apiserver` requests, breakdown of `apiserver` requests by HTTP method and response code, etc. For the complete list of supported metrics, see [API server data](/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#api-server-data).
+* **Scheduler**: requested CPU/memory vs available on the node, tolerations to taints, any set affinity or anti-affinity, etc. For the complete list of supported metrics, see [Scheduler data](/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#scheduler-data).
+* **Controller manager**: resident memory size, number of OS threads created, goroutines currently existing, etc. For the complete list of supported metrics, see [Controller manager data](/docs/integrations/kubernetes-integration/understand-use-data/understand-use-data#controller-manager-data).
+
+## Compatibility and requirements [#compatibility]
+
+* Control plane monitoring support is limited for managed clusters. This is because most cloud providers do not expose the metrics endpoints for the control plane components, so New Relic cannot access them.
+* When deploying the solution in [unprivileged mode](/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration#unprivileged), control plane setup will require [extra steps](#hostnetwork-privileged) and some caveats might apply.
+* [OpenShift](http://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG) 4.x uses control plane component metric endpoints that are different than the default.
+
+## Control plane component [#component]
+
+The task of monitoring the Kubernetes control plane is a responsibility of the `nrk8s-controlplane` component, which by default is deployed as a DaemonSet. This component is automatically deployed to master nodes, through the use of a default list of `nodeSelectorTerms` which includes labels commonly used to identify master nodes, such as `node-role.kubernetes.io/control-plane` or `node-role.kubernetes.io/master`. Regardless, this selector is exposed in the `values.yml` file and therefore can be reconfigured to fit other environments.
+
+Clusters that do not have any node matching these selectors will not get any pod scheduled, thus not wasting any resources and being functionally equivalent of disabling control plane monitoring altogether by setting `controlPlane.enabled` to `false` in the Helm Chart.
+
+Each component of the control plane has a dedicated section, which allows to individually:
+
+* Enable or disable monitoring of that component
+* Define specific selectors and namespaces for discovering that component
+* Define the endpoints and paths that will be used to fetch metrics for that component
+* Define the authentication mechanisms that need to be used to get metrics for that component
+* Manually specify endpoints that skip autodiscovery completely[^staticEndpoints]
```

Comment:
this looks odd in the preview

Reply:
🤦🏻‍♀️ 
I just googled that mdx supported markdown footnotes, and it turns out I should have googled if mdx _properly_ supports markdown footnotes.